docs/sources/release-notes/v2.5.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Grafana Mimir version 2.5 release notes"
+menuTitle: "V2.5 release notes"
+description: "Release notes for Grafana Mimir version 2.5"
+weight: 300
+---
+
+# Grafana Mimir version 2.5 release notes
+
+Grafana Labs is excited to announce version 2.5 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- Alertmanager discord support
+- TLS minimum version and cipher suites are now configurable for HTTP and gRPC servers
+- Various improvements to lower memory usage of store-gateway and ingester
+- New dashboards: "Mimir / Overview resources" and Mimir / Overview networking"
```

Comment:
I would rather add a single entry like "Improved Mimir dashboards" and then share a gist of the main improvements we did. They were several.

Reply:
How about this: https://github.com/grafana/mimir/pull/3580/files#diff-fddaf7aaf5533526b861513c83bb17e342adcfbec5e0c7c36bef8e0c47e49083R25-R29